### ISSUE TITLE
Dataset too similar?

### ISSUE THREAD

**Issue by ghost** [dusty-nv/jetson-inference#1203]
So I'm doing the retraining SSD and I built a dataset that includes Car, Truck, Motorcycle, Taxi, and Bus. I noticed after 30 epochs that my validation loss has kind of plateaued at around 2.8. My data set looked like:
![image](https://user-images.githubusercontent.com/86692714/130891223-1402876b-5259-4e83-bc9d-58599291a7f3.png)

After the first 30 epochs I decided to get another dataset to get some different pictures in the mix (not sure if thats a good or bad idea) and I'm going to train for another 30 epochs using pretrained-ssd from the last checkpoint. I guess I'm just wondering if maybe my dataset is too similar. Maybe I just need to train for a while?
